test/mocha/mocha.css

```diff
@@ -132,6 +132,7 @@ body {
   color: #c00;
   max-height: 300px;
   overflow: auto;
+  white-space: pre-wrap;
```

Comment:
if so we should send a PR to https://github.com/mochajs/mocha and then pull it in after it got merged.